### DISCUSSION TITLE
The source string has changed meanwhile. Please check your changes.

### DISCUSSION THREAD

**Discussion by Ere-Eye** [WeblateOrg/weblate#11605] [Q&A]
![image](https://github.com/WeblateOrg/weblate/assets/14802003/97cfa27f-9ef5-4f68-8be4-100137e7871b)
Multiple people have been reporting their changes not being saved in Weblate. I believe this error is the culprit. I can assure no one is touching the source strings, ever. I think it's caused by multiple people working on the same component at the same time and that changes the offset of state:<translated and pressing "Save and continue" causes to lose changes made in the translation. My hypothesis is further strengthened by the fact that "Save and continue" leads to another string is that is not adjacent to the previous one and pressing the left arrow doesn't go back to the string the user was facing just before. This problem is especially painful when it comes to the really long pieces of text.

**Comment by Ere-Eye** [comment]
@nijel This also happens when I'm away from the keyboard for extended periods of time. Upon coming back and "Save and continue", I get this exact same message even though the source string hasn't changed.

**Comment by nijel** [comment]
This can happen when server side cached results expire while translating. The proper fix is planned in https://github.com/WeblateOrg/weblate/issues/9953.
